sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -261,6 +261,17 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val DISK_TO_MEMORY_SIZE_FACTOR = buildConf(
+    "org.apache.spark.sql.execution.datasources.sizeFactor")
```

Comment:
`...sizeFactor` is too vague, how about `fileDataSizeFactor`?

Reply:
done